### ISSUE TITLE
Add CutefishDE for Ubuntu

### ISSUE THREAD

**Issue by whyadamsalt** [cutefishos/core#4]
I personally feel like a PPA for the cutefishDE would be very useful and appreciated as it will open up the desktop to further testing on Ubuntu based systems as well as allowing more users to be able to access the DE.

**Comment by telmo1241** [comment]
in my opinion a deb would be better right away so that it can have compatibility not only with Ubuntu, but with the debian base

**Comment by whyadamsalt** [comment]
A .deb file for a Desktop Environment doesn't seem logical. If you mean installing via APT then you'd need to wait for the next release of Ubuntu and Debian for it to be in official repositories.

**Comment by telmo1241** [comment]
I see, but in that case there should also be a copr for fedora

**Comment by whyadamsalt** [comment]
Yeah I agree. A copr for fedora and a ppa for Ubuntu would be ideal :)

**Comment by N0rbert** [comment]
Please prepare official PPA for CuteFish. On Ubuntu MATE QA we have https://github.com/ubuntu-mate/daily-brew/tree/main/.github/workflows which uploads packages to https://launchpad.net/~ubuntu-mate-bot/+archive/ubuntu/daily-brew . I'm not familiar with the procedure, but you can do the same for CuteFish. I'll suggest to target at 21.10 and upcoming 22.04 LTS while creating deb-packages.
We (I and @apandada1) have already some "progress" on packaging for Ubuntu - see https://askubuntu.com/a/1346398/66509 . But for me it was volunteered single time interest.
